smalltalksrc/VMMaker/SpurGenerationScavenger.class.st

```diff
@@ -1279,15 +1283,15 @@ SpurGenerationScavenger >> scavengeUnfiredEphemeronsOnEphemeronList [
 ]
 
 { #category : #accessing }
-SpurGenerationScavenger >> scavengerTenuringThreshold [ "(Slang flattens so need unique selectors)"
+SpurGenerationScavenger >> scavengerTenuringProportion [ "(Slang flattens so need unique selectors)"
```

Comment:
I think we can get rid of the comment `"(Slang flattens so need unique selectors)"` right?
That's kind of valid for most of the code :)